configuring.tex

```diff
@@ -97,6 +97,25 @@ \section{Preparing for First Use: Formatting SD Cards}
 Record (``Boot Sector'') by instead typing ``FIX MBR,'' should the
 need arise.
 
+
+As an end result, we want to have a correctly formatted SD card with the essential files stored on it for MEGA65 to boot from.
+
+This is how it works: When powering on, MEGA65 will search for- and boot these files:
+o FREEZER.M65
+o AUDIOMIX.M65
+o C64THUMB.M65
+o C65THUMB.M65
+o MEGA65.ROM
+o MEGA65.D81 (default disk image, automatically mounted during start)
+
+Straight out-of-the-box, MEGA65 will only have one SD card installed, located inside behind the trap-door. This SD card contains the essential files to properly boot from.
+When an external microSD card is inserted, MEGA65 consider the external one as higher priority and will try to boot from it.
+That means that the microSD card needs to have the essential files on it, otherwise MEGA65 cannot boot properly and will fall back into loading the OpenROM which does not support all the MEGA65 features.
+In general, if MEGA65 cannot boot properly and fall back to OpenROM, it will be shown in the power-up screen, similar to this:
+
+\includegraphics[width=\linewidth]{images/mega65-OpenROM boot-noSD.png}
```

Comment:
As we discussed, you wil need to make sure there are no spaces in the file name. The -'s are ok, though.

Reply:
Done